src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.additional-information-icon {
+  border: 1px solid var(--palette-grey-light);
+  border-radius: 50%;
+  color: var(--palette-grey-dark);
+  cursor: pointer;
```

Comment:
I'm not sure this is a valid application of `pointer`. From the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#Values):

> The cursor is a pointer that indicates a link. Typically an image of a pointing hand.

Reply:
Re 1:1 discussion, we should review this with design.